docs/getting-started-guides/gce.md

```diff
@@ -23,7 +23,7 @@ The example below creates a Kubernetes cluster with 4 worker node Virtual Machin
 
 ### Before you start
 
-If you want a simplified getting started experience and GUI for managing clusters, please consider trying [Google Container Engine](https://cloud.google.com/container-engine/) for hosted cluster installation and management.
+If you want a simplified getting started experience and GUI for managing clusters, please consider trying [Google Container Engine](https://cloud.google.com/container-engine/) (GCE) for hosted cluster installation and management.
```

Comment:
exactly my point here about "Compute" vs "Container" :-)

(not picking on you--just that this is hard to visually diff quickly!)

anyway this should be GKE